documentation/release_notes.markdown

```diff
@@ -14,6 +14,82 @@ canonical: "/puppetdb/latest/release_notes.html"
 [metrics]: ./api/metrics/v1/changes-from-puppetdb-v3.html
 [pqltutorial]: ./api/query/tutorial-pql.html
 
+4.1.0
+-----
+
+PuppetDB 4.1.0 is a backward-compatible feature release that adds packages for
+Ubuntu Xenial - 16.04, adds preliminary sorry for HUP signal handling, and
+improves the speed of removing old reports and node expiration and purging.
+
+### New features / Enhancements
+
+* PuppetDB now has packages for Ubuntu Xenial - 16.04
+  ([PDB-2475](https://tickets.puppetlabs.com/browse/PDB-2475)).
+
+* Allow configuration of AciveMQ Broker's memoryLimit. For PuppetDB instances
+  with larger amount of memory and heavy load, this can improve performance.
+  More information in the [config docs][./configure.html#memory-usage]
+  ([PDB-2726](https://tickets.puppetlabs.com/browse/PDB-2726)).
+
+* Preliminary support for HUP signal handling. Note that due to
+  [AMQ-5263][https://issues.apache.org/jira/browse/AMQ-5263] there is a
+  possibility of a crash when HUPed. We've observed this typically happens when
```

Comment:
I think I'll remove the "typically" here. It makes it sound like it happens more than it really does.